### PR TITLE
Parity with testcontainers-java

### PR DESCRIPTION
**Pull request by LMnet**
Fix for #95, #80

Most of the changes here are pretty mechanical. But some containers were pretty complex.

I didn't add tests. This is not good in general, but most of the containers are so simple and straightforward so I think it could be done in the future when some issues will be found. 

A detailed review of changes you can find in the changelog section of the readme.

### CODE REVIEW

**Comment by dimafeng** [comment]
@LMnet wow!

**Comment by LMnet** [comment]
@dimafeng ready for the review :)

File: modules/clickhouse/src/main/scala/com/dimafeng/testcontainers/ClickHouseContainer.scala [hunk 1]
```diff
@@ -0,0 +1,38 @@
+package com.dimafeng.testcontainers
+
+import org.testcontainers.containers.{ClickHouseContainer => JavaClickHouseContainer}
+
+class ClickHouseContainer(
```
Comment (dimafeng) [inline modules/clickhouse/src/main/scala/com/dimafeng/testcontainers/ClickHouseContainer.scala]: I think we need to have all classes declared as case classes for consistency

**Comment by dimafeng** [comment]
@LMnet seems the build is failing due to 
```
[pool-1-thread-1] INFO 🐳 [docker-compose] - Docker Compose has finished running

[info] Compiling 1 Scala source to /home/travis/build/testcontainers/testcontainers-scala/modules/db2/target/scala-2.11/classes ...

[info] Done compiling.

[info] Compiling 1 Scala source to /home/travis/build/testcontainers/testcontainers-scala/modules/postgres/target/scala-2.11/classes ...

[info] Done compiling.

[info] Compiling 1 Scala source to /home/travis/build/testcontainers/testcontainers-scala/modules/mysql/target/scala-2.11/classes ...

[info] Done compiling.

[info] Compiling 1 Scala source to /home/travis/build/testcontainers/testcontainers-scala/modules/oracle/target/scala-2.11/classes ...

[info] Done compiling.

[info] Compiling 1 Scala source to /home/travis/build/testcontainers/testcontainers-scala/modules/mssqlserver/target/scala-2.11/classes ...

[error] /home/travis/build/testcontainers/testcontainers-scala/modules/mssqlserver/src/main/scala/com/dimafeng/testcontainers/MSSQLServerContainer.scala:8:1: identifier expected but ')' found.

[error] ) extends SingleContainer[JavaMSSQLServerContainer[_]] with JdbcDatabaseContainer {

[error] ^

[error] /home/travis/build/testcontainers/testcontainers-scala/modules/mssqlserver/src/main/scala/com/dimafeng/testcontainers/MSSQLServerContainer.scala:38:2: ':' expected but eof found.

[error] }

[error]  ^

[error] two errors found

[info] Compiling 1 Scala source to /home/travis/build/testcontainers/testcontainers-scala/modules/clickhouse/target/scala-2.11/classes ...

[error] /home/travis/build/testcontainers/testcontainers-scala/modules/clickhouse/src/main/scala/com/dimafeng/testcontainers/ClickHouseContainer.scala:8:1: identifier expected but ')' found.

[error] ) extends SingleContainer[JavaClickHouseContainer] with JdbcDatabaseContainer {

[error] ^

[error] /home/travis/build/testcontainers/testcontainers-scala/modules/clickhouse/src/main/scala/com/dimafeng/testcontainers/ClickHouseContainer.scala:38:2: ':' expected but eof found.

[error] }

[error]  ^

[error] two errors found

[info] Compiling 1 Scala source to /home/travis/build/testcontainers/testcontainers-scala/modules/mariadb/target/scala-2.11/classes ...

[info] Done compiling.

[info] Compiling 1 Scala source to /home/travis/build/testcontainers/testcontainers-scala/modules/cockroachdb/target/scala-2.11/classes ...

[error] /home/travis/build/testcontainers/testcontainers-scala/modules/cockroachdb/src/main/scala/com/dimafeng/testcontainers/CockroachContainer.scala:7:1: identifier expected but ')' found.

[error] ) extends SingleContainer[JavaCockroachContainer] with JdbcDatabaseContainer {

[error] ^

[error] /home/travis/build/testcontainers/testcontainers-scala/modules/cockroachdb/src/main/scala/com/dimafeng/testcontainers/CockroachContainer.scala:30:2: ':' expected but eof found.

[error] }

[error]  ^

[error] two errors found
```

File: modules/clickhouse/src/main/scala/com/dimafeng/testcontainers/ClickHouseContainer.scala (hunk 1 above)
Comment (LMnet) [reply modules/clickhouse/src/main/scala/com/dimafeng/testcontainers/ClickHouseContainer.scala]: Done

**Comment by LMnet** [comment]
Fixed errors

**Pull request merged**